containers/python/PythonContainerDockerfile

```diff
@@ -3,7 +3,7 @@ FROM clipper/py-rpc:latest
 MAINTAINER Dan Crankshaw <dscrankshaw@gmail.com>
 
 COPY python_container.py python_container_entry.sh /container/
-COPY pywrencloudpickle.py python_container_conda_deps.txt /lib/
+COPY ../../clipper_admin/pywrencloudpickle.py python_container_conda_deps.txt /lib/
```

Comment:
This won't work. All the paths copied into the container need to be in the directory tree whose root is the directory containing the Dockerfile (https://docs.docker.com/engine/reference/commandline/build/#build-with-path). This is also important to make sure that the automated builds on Docker hub work. Probably the easiest thing to do is to move all the container dockerfiles to the top level of the directory. You'll need to modify the paths in the Dockerfiles, as well as the script that builds all the files.